x-pack/platform/plugins/shared/fleet/public/applications/integrations/sections/epm/screens/installed_integrations/hooks/use_installed_integrations_actions.tsx

```diff
@@ -141,6 +145,89 @@ export function useInstalledIntegrationsActions() {
     [toasts, queryClient, setPollingBulkActions]
   );
 
+  const bulkRollbackIntegrations = useCallback(
+    async (
+      items: InstalledPackageUIPackageListItem[],
+      onActionCompleted?: (status: string) => void
+    ) => {
+      try {
+        setActionCompletedCallback(() => onActionCompleted);
+
+        const res = await sendBulkRollbackPackagesForRq({
+          packages: items.map((item) => ({ name: item.name })),
+        });
+
+        setPollingBulkActions((actions) => [
+          ...actions,
+          {
+            taskId: res.taskId,
+            type: 'bulk_rollback',
+            integrations: items,
+          },
+        ]);
+        toasts.addInfo({
+          title: i18n.translate(
+            'xpack.fleet.epmInstalledIntegrations.bulkActions.bulkRollbackInProgressTitle',
+            {
+              defaultMessage: 'Rollback in progress',
+            }
+          ),
+          content: i18n.translate(
+            'xpack.fleet.epmInstalledIntegrations.bulkActions.bulkRollbackInProgressDescription',
+            {
+              defaultMessage:
+                'The integrations and the policies are rolling back to the previous version.',
```

Comment:
I think `Selected integrations and policies are rolling back to the previous version` would sound better